conftest.py

```diff
@@ -15,5 +15,5 @@
 #
 import pytest
 
-pytest_plugins = ["jupyter_server.pytest_plugin"]
+#pytest_plugins = ["jupyter_server.pytest_plugin"]
```

Comment:
What is the deal about this again?  This is the second time this has surfaced in a pull request and I wonder if we need to make a different change.

Reply:
oh dag nabbit... yeah so I diable this locally when I run pytest ....something.  Havent had time to look into it.
It throws a value error stating the plugin is already registered. 
`raceback (most recent call last):
  File "/Users/akchin/opt/anaconda3/envs/dev/bin/pytest", line 11, in <module>
    sys.exit(console_main())
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 185, in console_main
    code = main()
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 143, in main
    config = _prepareconfig(args, plugins)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 319, in _prepareconfig
    pluginmanager=pluginmanager, args=args
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/manager.py", line 87, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 203, in _multicall
    gen.send(outcome)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/helpconfig.py", line 100, in pytest_cmdline_parse
    config: Config = outcome.get_result()
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1003, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1283, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1192, in _preparse
    early_config=self, args=args, parser=self._parser
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/manager.py", line 87, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1070, in pytest_load_initial_conftests
    self.pluginmanager._set_initial_conftests(early_config.known_args_namespace)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 503, in _set_initial_conftests
    self._try_load_conftest(current, namespace.importmode)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 508, in _try_load_conftest
    self._getconftestmodules(anchor, importmode)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 536, in _getconftestmodules
    mod = self._importconftest(conftestpath, importmode)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 587, in _importconftest
    self.consider_conftest(mod)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 663, in consider_conftest
    self.register(conftestmodule, name=conftestmodule.__file__)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 443, in register
    self.consider_module(plugin)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 669, in consider_module
    self._import_plugin_specs(getattr(mod, "pytest_plugins", []))
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 676, in _import_plugin_specs
    self.import_plugin(import_spec)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 713, in import_plugin
    self.register(mod, modname)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 436, in register
    ret: Optional[str] = super().register(plugin, name)
  File "/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/pluggy/manager.py", line 106, in register
    % (plugin_name, plugin, self._name2plugin)
ValueError: Plugin already registered: jupyter_server.pytest_plugin=<module 'jupyter_server.pytest_plugin' from '/Users/akchin/opt/anaconda3/envs/dev/lib/python3.7/site-packages/jupyter_server/pytest_plugin.py'>
`